.flake8

```diff
@@ -0,0 +1,4 @@
+[flake8]
+max-complexity = 10
+max-line-length = 119
+exclude = .venv,djstripe/migrations/,tests,docs
```

Comment:
I'd rather have config vars in the setup.cfg file. Is there a reason this needed to be broken out?